src/sage/geometry/integral_points_integer_dense.pyx

```diff
@@ -0,0 +1,6 @@
+#cython: wraparound=False, boundscheck=False
```

Comment:
To my understanding, this will just not build, if flint is not present.

From my quick "research" of the state of things, this file will not be included in a sage polyhedra distribution, if flint is not present?

Reply:
Which modules are included in any distribution is static. 
This module is simply not included in **sagemath-polyhedra** and will be provided by a different distribution.